### PR TITLE
DOCSP-41050-Compass-1.43.4

### PR DESCRIPTION
**Pull request by ajhuh-mdb**
## DESCRIPTION
Compass 1.43.4 release notes

## STAGING
https://preview-mongodbajhuhmdb.gatsbyjs.io/compass/DOCSP-41050-compass-1.43.3/release-notes/

## JIRA
https://jira.mongodb.org/browse/DOCSP-41050

## BUILD LOG
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=66830561fe162e5006d980bc
## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/release-notes.txt [hunk 1]
```diff
@@ -10,6 +10,23 @@ Release Notes
...
 
+|compass| 1.43.4
+----------------
+
+*Released July 1, 2024*
+
+Bug Fixes:
+
+- Prevents application from hanging when selecting ranges too quickly on the 
+  "Schema" tab (:issue:`COMPASS-8048`).
```
Comment (jeff-allen-mongo) [inline source/release-notes.txt:21]: [Optional suggestion]

Maybe link "Schema tab" to this page: https://www.mongodb.com/docs/compass/current/schema/

File: source/release-notes.txt [hunk 2]
```diff
@@ -10,6 +10,23 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.43.4
```
Comment (jeff-allen-mongo) [inline source/release-notes.txt:13]: Let's also update the version in the snooty.toml file for this release.

**Review by jeff-allen-mongo — APPROVED** [review]
LGTM % nits. Thanks @ajhuh-mdb!

File: source/release-notes.txt (hunk 2 above)
Comment (ajhuh-mdb) [reply source/release-notes.txt:13]: thanks for catching that! updating.

**Comment by ajhuh-mdb** [comment]
Hi @gribnoysup 👋 Whenever you can, could you review these release notes for Compass 1.34.4? TIA!

**Review by gribnoysup — APPROVED** [review]
Looks good, thanks!

**Pull request merged**